client/packages/system/src/Asset/ImportCatalogueItem/UploadTab.tsx

```diff
@@ -121,7 +117,7 @@ export const AssetItemUploadTab: FC<ImportPanel & AssetItemUploadTabProps> = ({
   const { error } = useNotification();
   const [isLoading, setIsLoading] = useState(false);
   const AssetItemBuffer: AssetItemImportModal.ImportRow[] = [];
-  const { data: properties } = useAssetData.utils.properties();
+  const { data: properties } = useAssetData.utils.properties(undefined);
```

Comment:
would be easier to read if 
(a) the filter prop was optional and defaulted to `undefined`, or
(b) was a named property and so would read here as `({filter: undefined})`

Reply:
[e71dd1e](https://github.com/msupply-foundation/open-msupply/pull/4001/commits/e71dd1e4b80710f4bbf81ff70a0ec6cbe997bfd6)